### DISCUSSION TITLE
Issue Tasks

### DISCUSSION THREAD

**Discussion by Malix-Labs** [community/community#19502] [Projects and Issues]
# Issue Tasks

## Problem

About issues descriptions:
- **Markdown checkboxes are always considered as tags** _(from #5238)_
- **Tasks cannot have their own issue proprieties** _(such as differentiate assignees, projects, reviewers, and etc for each tasks)_

## Solution

Considering issue task as sub-issues as a whole object

**Comment by thejeff77** [comment]
https://github.com/orgs/community/discussions/5238#discussioncomment-3703479
